<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnSolo">
      <property name="text">
       <string>JOGAR (SOLO)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHost">
      <property name="text">
       <string>HOST</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnJoin">
      <property name="text">
       <string>ENTRAR</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBg">
      <property name="text">
       <string>BACKGROUND</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRestoreBg">
      <property name="text">
       <string>RESTAURAR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
